### PR TITLE
Fix package metadata

### PR DESCRIPTION
**Pull request by Eilon**
@ryanbrandenburg @DamianEdwards

### CODE REVIEW

File: src/Microsoft.AspNetCore.Localization/project.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
   "version": "1.0.0-*",
-  "description": "Middleware for automatically applying culture information to HTTP requests.",
+  "description": "ASP.NET Core middleware for automatically applying culture information to HTTP requests. Culture information can be specified in the HTTP header, query string, cookie, or custom source.",
```
Comment (ryanbrandenburg) [inline src/Microsoft.AspNetCore.Localization/project.json]: Do we really want to get so specific about how culture information can be specified here? Seems like a deeper dive that might more appropriate for docs to me, and like a thing we would forget to update if we added or removed ways to specify localization.

File: src/Microsoft.Extensions.Localization.Abstractions/project.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {
   "version": "1.0.0-*",
-  "description": "Abstractions of application localization services.",
+  "description": "Abstractions of application localization services.\r\nCommonly used types:\r\nMicrosoft.Extensions.Localization.IStringLocalizer\r\nMicrosoft.Extensions.Localization.IStringLocalizer<T>",
```
Comment (ryanbrandenburg) [inline src/Microsoft.Extensions.Localization.Abstractions/project.json]: Is "Commonly used types" a convention we follow for all Abstractions packages?

Comment (Eilon) [inline src/Microsoft.Extensions.Localization.Abstractions/project.json]: It's something I've been doing when there are some especially interesting types in a package. CoreFX team has been doing this and it seems useful to me. It's not specific to abstractions packages.

File: src/Microsoft.AspNetCore.Localization/project.json (hunk 1 above)
Comment (Eilon) [inline src/Microsoft.AspNetCore.Localization/project.json]: I certainly hope we never remove any of these ways - that would be a breaking change. And if we add them, the comment remains correct. The goal is for the description to be sufficient to understand what's in the package and why you'd care to install it. Or, similarly, to understand whether it's ok to remove a reference to the package.

File: src/Microsoft.Extensions.Globalization.CultureInfoCache/project.json [hunk 3]
```diff
@@ -1,6 +1,10 @@
 {
   "version": "1.0.0-*",
   "description": "Provides cached instances of CultureInfo using a generated list of known culture names for use in scenarios where unbounded CultureInfo creation is undesirable.",
+  "tags": [
+    "globalization",
```
Comment (muratg) [inline src/Microsoft.Extensions.Globalization.CultureInfoCache/project.json]: Localization project.json also has "aspnetcore" in the tags. Consider adding it here too.

Comment (Eilon) [inline src/Microsoft.Extensions.Globalization.CultureInfoCache/project.json]: This one isn't for ASP.NET (no AspNet in the name).

**Pull request merged**